host/xtest/sdp_basic.c

```diff
@@ -76,6 +78,37 @@ struct tee_ctx {
 	TEEC_Session sess;
 };
 
+/*
+ * Old ION API to allocate and export a buffer
+ */
+static int allocate_ion_old_buffer(size_t size, int heap_type_id, int ion)
```

Comment:
Nitpicking: `allocate_old_ion_buffer()`?

Reply:
that's right. i'll fix.